client/.eslintrc.json

```diff
@@ -0,0 +1,17 @@
+{
+  "root": true,
+  "parser": "@typescript-eslint/parser",
+  "plugins": [
+    "@typescript-eslint", "react", "react-hooks"
+  ],
+  "extends": [
```

Comment:
complete the prettier integration also her, and add the rules that you have added it in backend config